examples/build.gradle

```diff
@@ -43,7 +43,7 @@ dependencies {
 protobuf {
     protoc { artifact = "com.google.protobuf:protoc:${protocVersion}" }
     plugins {
-        grpc { artifact = "io.grpc:protoc-gen-grpc-java:${grpcVersion}" }
+        grpc { artifact = "io.grpc:protoc-gen-grpc-java:1.54.0" }
```

Comment:
You probably didn't mean to leave this change in?

Reply:
Undid the change